src/libs/geotagging.c

```diff
@@ -863,23 +1679,132 @@ void gui_init(dt_lib_module_t *self)
+                                "\n- the number of images going to be geotagged "
+                                "based on images date time, offset and time zone"
+                                "\n- more detailled time information hovering the row"));
+  renderer = gtk_cell_renderer_toggle_new();
+  g_signal_connect(renderer, "toggled", G_CALLBACK(_track_seg_toggled), self);
+  GtkTreeViewColumn *column = gtk_tree_view_column_new_with_attributes("", renderer, "active", DT_GEO_TRACKS_ACTIVE, NULL);
+  gtk_tree_view_append_column(GTK_TREE_VIEW(d->map.gpx_view), column);
+  d->map.sel_tracks = column;
+  GtkWidget *button = gtk_check_button_new();
+  gtk_widget_show(button);
+  gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(button), FALSE);
+  gtk_tree_view_column_set_widget(column, button);
+  gtk_tree_view_column_set_alignment(column, 0.5);
+  g_signal_connect(column, "clicked", G_CALLBACK(_all_tracks_toggled), self);
+
+  column = _new_tree_text_column(_("start time"), TRUE, 0.0, DT_GEO_TRACKS_DATETIME, PANGO_ELLIPSIZE_START);
+  gtk_tree_view_append_column(GTK_TREE_VIEW(d->map.gpx_view), column);
+  column = _new_tree_text_column(_("points"), FALSE, 1.0, DT_GEO_TRACKS_POINTS, PANGO_ELLIPSIZE_NONE);
+  gtk_tree_view_append_column(GTK_TREE_VIEW(d->map.gpx_view), column);
+  column = _new_tree_text_column(_("images"), FALSE, 1.0, DT_GEO_TRACKS_IMAGES, PANGO_ELLIPSIZE_NONE);
+  gtk_tree_view_append_column(GTK_TREE_VIEW(d->map.gpx_view), column);
+
+  g_object_set(G_OBJECT(d->map.gpx_view), "has-tooltip", TRUE, NULL);
+  g_signal_connect(G_OBJECT(d->map.gpx_view), "query-tooltip", G_CALLBACK(_row_tooltip_setup), self);
+
+  // avoid ugly console pixman messages due to headers
+  gtk_tree_view_set_headers_visible(GTK_TREE_VIEW(d->map.gpx_view), FALSE);
+  GtkWidget *w = dt_ui_scroll_wrap(GTK_WIDGET(d->map.gpx_view), 100, "plugins/lighttable/geotagging/heighttracklist");
+  gtk_widget_set_size_request(w, -1, 100);
+  gtk_tree_view_set_headers_visible(GTK_TREE_VIEW(d->map.gpx_view), TRUE);
+  gtk_box_pack_start(GTK_BOX(d->map.gpx_section), w, TRUE, TRUE, 0);
+
+  grid = GTK_GRID(gtk_grid_new());
+  gtk_grid_set_column_spacing(grid, DT_PIXEL_APPLY_DPI(5));
+  line = 0;
+
+  d->map.preview_button = gtk_check_button_new_with_label(_("preview images"));
+  gtk_toggle_button_set_active(GTK_TOGGLE_BUTTON(d->map.preview_button), TRUE);
+  gtk_widget_set_sensitive(d->map.preview_button, FALSE);
+  gtk_label_set_ellipsize(GTK_LABEL(gtk_bin_get_child(GTK_BIN(d->map.preview_button))), PANGO_ELLIPSIZE_END);
+  gtk_grid_attach(grid, d->map.preview_button, 0, line, 1, 1);
+  gtk_widget_set_tooltip_text(GTK_WIDGET(d->map.preview_button),
+                              _("show on map images going to be geotagged"));
+  g_signal_connect(GTK_TOGGLE_BUTTON(d->map.preview_button), "toggled", G_CALLBACK(_images_preview_toggled), self);
+
+  d->map.select_button = dt_ui_button_new(_("select images"), _("select images going to be geotagged"), NULL);
```

Comment:
maybe just:

"select images to be geotagged"

Reply:
slightly not the same meaning...
What I've set means (or at least it's my intention): images compatible with gpx file and which will be geotagged if one continues the process.
What you have written may not be accurate if there are more selected images than compatible images with the gpx file.

What do you think ? If you persist I'll do the change.

Reply:
"select images going to be geotagged" is not proper English I think, it should be "select images which are going to be geotagged" (if that what you meant), but to me it has exactly the same meaning as "select images to be geotagged". I don't understand the difference you're talking about. Do you mean "select images to try applying geotag to" (which sounds ugly)? I don't think this difference matters.

Reply:
> "select images which are going to be geotagged" (if that what you meant)

Correct.
I do see a difference but my English is not accurate enough to discuss more about this.
I'll do the change.